packages/esm-appointments-app/src/patient-appointments/patient-appointments-header.scss

```diff
@@ -0,0 +1,37 @@
+@use '@carbon/styles/scss/type';
```

Comment:
copied all these styles from some styling for the patient banner in the patient chart app